pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/buffer/impl/InMemTransactionBuffer.java

```diff
@@ -347,4 +347,19 @@ private void addTxnToTxnIdex(TxnID txnId, long committedAtLedgerId) {
         return CompletableFuture.completedFuture(null);
     }
 
+    @Override
+    public boolean isTxnAborted(TxnID txnID) {
+        return false;
+    }
+
+    @Override
+    public void syncMaxReadPositionForNormalPublish(PositionImpl position) {
+        //no-op
+    }
+
+    @Override
+    public PositionImpl getMaxReadPosition() {
+        return PositionImpl.latest;
```

Comment:
Why the in-memory transaction buffer returns latest?

Reply:
in-memory didn't need stable position logical, I don't change it.